LPS-157657 String XML appears with defined Description or Title at Title Field on System Object Commerce Product Group

ces-locpham opened the pull request:
Hi @thienquach87 , 
Please help me review this request.
This [ticket](https://issues.liferay.com/browse/LPS-157657)
Thanks

cgoncas committed:
LPS-158043 Add SUBSCRIBE and UNSUBSCRIBE actions to the API

cgoncas committed:
LPS-158043 Send Subscribe information to the frontend.

cgoncas committed:
LPS-158043 Implement Subscribe/Unsubscribe for Journal Articles.

cgoncas committed:
LPS-158043 Add bell to the frontend.

brookedalton committed:
LRQA-74893 Add testcase to assert alias case insensitivity

cgoncas committed:
LPS-148882 Create vocabularies every time a new service is registered.

Stephy6 committed:
LPS-148882 Create enum class for Internal Vocabularies constants

ling-alan-huang committed:
LPS-150378 Catch more cases, but we don't do inline when the variable is in a methoed call and in arithmetic

cgoncas committed:
LPS-148882 New AssetVocabularyUtil that extract the common code for adding  new asset vocabularies

cgoncas committed:
LPS-148882 Use the new Util

DiegoHu97 committed:
LPS-155625 Give feedback to the user when they cannot save a fragment composition

ling-alan-huang committed:
LPS-150378 SF, inline

ces-locpham committed:
LPS-156222 The required attribute does not work for the LocalizedText field type

amosfong committed:
LPS-157950 add object relationships

ealonso committed:
LPS-159597 portal-workflow-task-web: Uses liferay-ui:icon instead of liferay-frontend:management-bar-button, since it is for management toolbar buttons

ealonso committed:
LPS-159597 portal-workflow-web: Uses liferay-ui:icon instead of liferay-frontend:management-bar-button, since it is for management toolbar buttons

Lucas2691 committed:
LPS-158459 Fix date inconsistency

Lucas2691 committed:
LPS-158459 SF

Lucas2691 committed:
LPS-158459 Set logic date to show year

Lucas2691 committed:
LPS-158459 SF

ces-locpham committed:
LPS-157657 String XML appears with defined Description or Title at Title Field on System Object Commerce Product Group

Lucas2691 committed:
LPS-158459 Update code to fit date with locale

Lucas2691 committed:
LPS-158459 SF

Lucas2691 committed:
LPS-158459 Adjusting pages to pass locale

ling-alan-huang committed:
LPS-150378 Don't do inline when the variable definition has annotation

ling-alan-huang committed:
LPS-150378 Add suppression

ling-alan-huang committed:
LPS-150378 SF, inline

ling-alan-huang committed:
LPS-150378 SF

ling-alan-huang committed:
LPS-150378 Don't do inline when the variable is used in for or while conditons

thienquach87 commented on modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:
```diff
@@ -610,11 +611,19 @@ public Map<String, Object> getSystemModelAttributes(
+					ObjectFieldConstants.DB_TYPE_STRING)) {
+
+				value = LocalizationUtil.getLocalization(
+					(String)value, null, true);
```
to make code cleaner, should reference to the below code-lines
if( value instanceof String ) {
	value = LocalizationUtil.getLocalization((String)value, null, true);
}

ces-locpham replied:
I fixed follow this comment @thienquach87

ealonso committed:
LPS-159597 export-import-changeset-taglib: Uses liferay-ui:icon instead of liferay-frontend:management-bar-button, since it is for management toolbar buttons

ealonso committed:
LPS-159597 Unused methods

pei-jung committed:
LPS-159104 Only show the account address tab if user has permission

pei-jung committed:
LPS-159104 Check for MANAGE_ADDRESSES permission

carolmariaabb committed:
LPS-158827 Simplify

carolmariaabb committed:
LPS-158827 Fix integration tests

Lucas2691 committed:
LPS-158459 Get locate from Liferay Language

Lucas2691 committed:
LPS-158459 Add dateCreated to unit test

alberttgomes committed:
LPS-139478 creation of new PageReset file to check if there are fields on that page with rules applied

guilhermedcamacho committed:
LPS-159400 Move class SalesforceConfiguration to api module

guilhermedcamacho committed:
LPS-159400 Create integration tests for class SalesforceObjectEntryManagerImpl

ling-alan-huang committed:
LPS-159401 No need to get archived module directory names outside portal source

ling-alan-huang committed:
LPS-159401 Catch more cases

ling-alan-huang committed:
LPS-159401 Always format poshi.properties even it's in other reop

lfbesada committed:
LPS-158149 Adds a class to assert UpdateFormItemConfigMVCActionCommand

cgoncas committed:
LPS-152942 Delete FF

sandrodw3 committed:
LPS-159414 Move hasFormParent to utils

cgoncas committed:
LPS-152942 Auto SF

ling-alan-huang committed:
LPS-159401 Improve performance

ling-alan-huang committed:
LPS-159401 Always format test.properties

sandrodw3 committed:
LPS-159414 Rename

sandrodw3 committed:
LPS-159414 Add util function to check if an item is required form input

ealonso committed:
LPS-138481 Uses serviceContext to set the create date in that way we are going to have the correct order

AliciaGarciaGarcia committed:
LPS-159176 Support field mapping for knowledge base components

ealonso committed:
LPS-152803 If we don't have order by keys then don't show the sorting icon

AliciaGarciaGarcia committed:
LPS-159176 add test to check the item provider function on knowledge base article

igor-franca committed:
LPS-147647 Remove scroll behavior on Number field type in Node Information

pei-jung committed:
LPS-122228 Register new tab for regions

pei-jung committed:
LPS-122228 Add SearchContainerFactory for regions

javalosjr96 committed:
LPS-159541 Bump org.mozilla:rhino to 1.7.14

DiegoHu97 committed:
LPS-155625 Add language keys

DiegoHu97 committed:
LPS-155625 buildLang

cgoncas committed:
LPS-152942 Baseline

magjed4289 committed:
LPS-159442 MessageErrorIsThrownWhenMethodNotImplemented

lfbesada committed:
LPS-158149 Adds an AutoCloseable implementation to handle InputsFragmentCollectionContributor component enabling and disabling

Co-authored-by: Jürgen Kappler <jurgen.kappler@liferay.com>

lfbesada committed:
LPS-158149 Add an integration test to assert that when the feature flag is disabled the form item configuration is updated properly but no items are added to the form

lfbesada committed:
LPS-158149 Add an integration test to assert that when the feature flag is enabled the form item configuration is updated properly and, in addition to the submit button, the proper item is added to the form for each editable info field

lfbesada committed:
LPS-158149 Extract to a method in order to reuse it

Jeremy-Ch committed:
LPS-141135 Use JSON to add entries

claraizquierdo committed:
LPS-158043 Semantic Versioning

ces-locpham committed:
LPS-156222 Update testEmpty for LocalizableTextDDMFormFieldValueAccessor

AliciaGarciaGarcia committed:
LPS-159176 Auto SF

lfbesada committed:
LPS-158149 Assert the resulting JSONObject

cgoncas committed:
LPS-152905 Add the preview action for blogs.

cgoncas committed:
LPS-152905 Fill the getPreview method with the right value.

cgoncas committed:
LPS-152905 Preview URL can be empty.

claraizquierdo committed:
LPS-158043 Sidebar.Header can render children

claraizquierdo committed:
LPS-158043 Move position of subscribe and unsubscribe icon

victorg1991 committed:
LPS-146373 Simplify SidebarHeaderPanel and add close button

victorg1991 committed:
LPS-146373 Adapt panels

sandrodw3 committed:
LPS-159414 Add util to get form validation data

Jeremy-Ch committed:
LPS-141135 Use the macro to go to click the edit tags icon

sandrodw3 committed:
LPS-159437 Show alert instead of modal when hidding required fragment

Jeremy-Ch committed:
LPS-141135 Update test to add entries by JSON

sandrodw3 committed:
LPS-159414 Show alert when deleting required fragment

sandrodw3 committed:
LPS-159437 We don't need formModalData anymore. Publish modal will be refactored in future subtask, that's why I'm still not using data from getFormWarningData

lfbesada committed:
LPS-158149 Add an integration test to assert that when updating other configuration values without changing the form mapping then the form content is preserved

lfbesada committed:
LPS-158149 Don't assert the fragmentEntryLink's rendererKey when the fragmentEntry wasn't added automatically because it could be any type of fragment and not necessarily an input OOTB fragment

lfbesada committed:
LPS-158149 SF extract constants

claraizquierdo committed:
LPS-152905 Fix preview properties

lfbesada committed:
LPS-158149 Assert the form content before invoke the action to ensure that the content is not set by the automation

lfbesada committed:
LPS-158149 Add an integration test to assert that when the form mapping is reset the form content is cleared out

lfbesada committed:
LPS-158655 Add an integration test to assert that when the OOTB InputsFragmentCollectionProvider is not available the server send the proper error message to the view

lfbesada committed:
LPS-158655 Add an integration test to assert that when can't find a fragmentEntry for an infoFieldType the server sends the proper error message to the view

lfbesada committed:
LPS-158655 Add required dependencies

Jeremy-Ch committed:
LPS-141135 Update test to add assets by JSON

lfbesada committed:
LPS-158655 SF extract to a field

lfbesada committed:
LPS-158655 Improves tests performance, use static final field

magjed4289 committed:
LPS-159442 Refactor of teardown

magjed4289 committed:
LPS-159442 SF

Commit:
LPS-159346 Get data from api to fill out applications quantity

alberttgomes committed:
LPS-139478 buildLanguage

pei-jung committed:
LPS-122228 Add mvc commands

guilhermedcamacho committed:
LPS-159400 Add test credentials

pei-jung committed:
LPS-122228 Implement view regions page

pei-jung committed:
LPS-122228 Use region's name if no localized name is defined

pei-jung committed:
LPS-122228 Implement edit region page

guilhermedcamacho committed:
LPS-159400 Add keys to not be printed

pei-jung committed:
LPS-122228 Show the back button on every tabs of the edit country page

pei-jung committed:
LPS-122228 Update warning messages

epark18 committed:
LRQA-76017 Add test CanConfigureEmailSettingsInVirtualInstance

ces-locpham committed:
LPS-156222 SF

Jeremy-Ch committed:
LPS-141135 Use loop to assert

Jeremy-Ch committed:
LPS-141135 Add documents by JSON

ealonso committed:
LPS-138481 Uses a new group and delete it after test execution

Jeremy-Ch committed:
LPS-141135 Update test to use the existing macro to edit tag

Jeremy-Ch committed:
LPS-141135 Use fast navigation to go to the folder

AliciaGarciaGarcia committed:
LPS-159176 test where not properly deleting the groups and where still on DB

claraizquierdo committed:
LPS-152905 The url is no necessary to display the asset preview

Jeremy-Ch committed:
LPS-141135 Use loop to add the entries

Jeremy-Ch committed:
LPS-141135 Add macro for the management bar navigation

Commit:
LPS-146373 Modify locators to adapt new markup

sandrodw3 committed:
LPS-158150 Add test for mapping from the form

victorg1991 committed:
LPS-146373 To move the scroll to the editor itself, set a fixed height for the wrapper

victorg1991 committed:
LPS-146373 Use margin to move the scrollbar

victorg1991 committed:
LPS-146373 Adapt TopperLabel now that the body is not the one scrolling

victorg1991 committed:
LPS-146373 Adapt tests

cgoncas committed:
LPS-158043 Add FF.

ccorreagg committed:
LPS-159215 Add pathContext when building paths from UriInfo

ccorreagg committed:
LPS-159215 Add test for UriInfoUtil

victorg1991 committed:
LPS-159618 Don't show lines if the drop is not valid

carolmariaabb committed:
LPS-158827 Revert "LPS-122340 Sort output json by attribute names for easier testing."

This reverts commit 646246f10fefcaec3de3973daf4154368434585c.

The field attribute value is serialized to be saved in the database. So to retrieve the data in its original form, it's enough to just deserialize the value

AliciaGarciaGarcia committed:
LPS-157505 you can not subscribe to a template but you can comment. prevent npe

AliciaGarciaGarcia committed:
LPS-157505 rename /admin/*jsp on knowledge base module

lfbesada committed:
LPS-159601 Use article version instead of version classPK

lfbesada committed:
LPS-159601 Treat version as a String

lfbesada committed:
LPS-159601 Set the infoItemIdentifier version

victoroscuro committed:
COMMERCE-9778 Fix search criteria and testcases

shuyangzhou committed:
LPS-159915 Unplug default ImageProcessorImpl, since the replacement AMImageEntryProcessor is always present.

ling-alan-huang committed:
LPS-150378 SF, inline

patriciajeanperez committed:
LRQA-76344 Use generic locators

patriciajeanperez committed:
LRQA-76344 Delete unused paths

patriciajeanperez committed:
LRQA-76345 Delete macros that are not being used

patriciajeanperez committed:
LRQA-76345 Use generic locator

patriciajeanperez committed:
LRQA-76345 Delete paths not being used

patriciajeanperez committed:
LRQA-76359 Use generic locators

patriciajeanperez committed:
LRQA-76359 Use generic locator for test

patriciajeanperez committed:
LRQA-76359 Delete paths not being used

Jeremy-Ch committed:
LPS-141135 Rename the tests

ericyanLr committed:
LPS-159458 Restore Read more link with Liferay Learn article

Previously removed by LPS-138008 (db019fd34772e748371cf5d3f52681c4d732c218)

Jeremy-Ch committed:
LPS-141135 Update tests to make it simple

sandrodw3 committed:
LPS-159414 buildLang

ealonso committed:
LPS-159597 baseline

victorg1991 committed:
LPS-159739 Fix CET definition

victorg1991 committed:
LPS-159739 Hide this for now because it cannot be used anywhere

claraizquierdo committed:
LPS-152905 Remove downloadURL from Preview

claraizquierdo committed:
LPS-152905 Fix preview test

claraizquierdo committed:
LPS-152905 Add new downloadURL field

claraizquierdo committed:
LPS-152905 Use new downloadURL field in SidebarPanelInfoView component

cgoncas committed:
LPS-148882 Use new constants.

Jeremy-Ch committed:
LPS-141135 Update the loop list parameter

Commit:
LPS-158043 Use Subscribe component in FE

Commit:
LPS-158043 Fix minor stuff in ManageCollaborators component

jgarcialr committed:
LPS-159766 compare against localized key instead of 'advanced' literal

lfbesada committed:
LPS-158040 Partial Revert "LPS-129091 asset-list-service: no reason for using the classTypeId when there is no structure id to form the ddmFieldName attribute"

This partial reverts commit 4a55c476494c20e99cb058db5118515f91c31b10.

Commit:
LPS-158043 Refetch data when subscribe

cgoncas committed:
LPS-152942 Use the right redirect.

lfbesada committed:
LPS-158040 Add a method to get the fieldReference

victoroscuro committed:
COMMERCE-9778 Fix macro and testcase failing due to Chrome 100

lfbesada committed:
LPS-158040 Use fieldReference instead of fieldName

ling-alan-huang committed:
LPS-159401 Avoid NoSuchFileException

ling-alan-huang committed:
LPS-159401 Apply SF check to all repos

Commit:
LPS-158043 Tooltip position, renaming and styles

ling-alan-huang committed:
LPS-159401 Improve logic to detect cases like fcbbba94af31912bd01e3b924de16113ef39a6bd

pei-jung committed:
LPS-122228 Refactor code to remove the use of cmd parameter

Kvale1733 committed:
LRQA-76371 Improve bookmark macros navigation to bookmark admin

Kvale1733 committed:
LRQA-76371 Fix PARENT_FOLDER_PANEL path

Kvale1733 committed:
LRQA-76371 Fix MERGE_WITH_PARENT_FOLDER path

Kvale1733 committed:
LRQA-76371 Port CPBookmarks.testcase to 7.4

pei-jung committed:
LPS-122228 Display error messages for region exceptions

kenjiheigel committed:
LRCI-2853 Only override github url when rebase is necessary

jgarcialr committed:
LPS-159766 simplify

Kvale1733 committed:
LRQA-76371 Port PGBookmarks.testcase to 7.4

kenjiheigel committed:
LRCI-2853 Add additional invocation parameters

shuyangzhou committed:
LPS-159915 Although this test is not testing real case, let's feed back the original ImageProcessorImpl to keep the logic the same to avoid massive rewrite for the test.

vicnate5 committed:
LRQA-76371 Update testcase file names to current standards

vicnate5 committed:
LRQA-76371 Include bookmarks in release

vicnate5 committed:
LRQA-76371 Test bookmarks in releases

liuxingye committed:
LPS-159493 Update test after LPS-134508

Commit:
LPS-159854 Modify locator to make it robust

austinchiang committed:
LPS-141135 Rename tests

Commit:
LPS-146373 Use suitable macro

Commit:
LPS-158537 Add step to increase the window width

Commit:
LPS-159747 Add locator for PAGE_VIEWPORT_PREVIEW

Commit:
LPS-159747 Add test to preview page when reviewing publication changes

jkappler committed:
LPS-159771 We always have the cookies set, so instead of checking for null, check if the userConfigCookieName is enabled or not. If it's not enabled, we show the configured values

ling-alan-huang committed:
LPS-159401 Move logic, no logical changes

victorg1991 committed:
LPS-146373 Round inset calculation to avoid errors

victorg1991 committed:
LPS-159739 baseline

ling-alan-huang committed:
LPS-159401 Generated

claraizquierdo committed:
LPS-152905 Fix FE tests

cgoncas committed:
LPS-158941 We should add the scopeGroupId to the theme display in order to read the blogsEntry in BlogsLayoutDisplayPageProvider.getLayoutDisplayPageObjectProvider after the changes in LPS-105422.

victorg1991 committed:
LPS-159444 Register the scheduler also when changing the instance settings

Commit:
LPS-158043 Add tests

Jeremy-Ch committed:
LPS-159857 Add step to expand the configuration panel

Jeremy-Ch committed:
LPS-159863 View the success message after saving the custom tags

Commit:
LPS-158043 SF

Commit:
LPS-158043 Fix localize aria label

liferay-translation committed:
LPS-77699 Update Translations

victoroscuro committed:
COMMERCE-9778 Update macro checkSuccessAlertAppearsAndDisappears

jgarcialr committed:
LPS-159875 remove HtmlUtil.escape. It's already scaped by autoescape annotation

victorg1991 committed:
LPS-159877 Show alert if there isn't any panel available

victorg1991 committed:
LPS-159877 Language key

victorg1991 committed:
LPS-159877 buildLang

jgarcialr committed:
LPS-159875 SF

cgoncas committed:
LPS-135167 Separate logic from Company and Group configuration.

cgoncas committed:
LPS-135167 Whe the scope is Company, we should review if the company scopePK is enabled.

cgoncas committed:
LPS-135167 Add tests.

Commit:
LPS-158043 Fix snapshots

ealonso committed:
LPS-159597 Adds data-title since it is used on poshi tests, also update poshi test path

silasbsi committed:
LPS-159022 Analysis of the data in the commission chart

Commit:
LPS-159781 Remove extension from file to assert

Commit:
LRAC-11607 Modify SegmentCriteriaOptionsContainsDuplicateAndDelete because use something related to accounts

brianchandotcom committed:
LPS-159215 Sort

brianchandotcom committed:
LPS-159215 You're updating path and scheme

rafaelfs98 committed:
LPS-159816 Bug fix for When listing categories in new options, options appear in place of categories

brianchandotcom committed:
LPS-159739 Don't remove it, just comment it out

brianchandotcom committed:
LPS-159346 upload

shuyangzhou committed:
LPS-159912 Always regsiter DependencyManagerSyncImpl to avoid callback misfire when dependency.manager.thread.pool.enabled is disabled.

magjed4289 committed:
LPS-159442 Solving conflicts

drewbrokke committed:
LPS-156222 dynamic-data-mapping-form-field-type: make sure the test will fail if the change is reverted

Commit:
LPS-159910 Fix CanCancelRenameColumnLabel test

anabuchmann committed:
LPS-159774 - Fix Workflow Metrics tests

cgoncas committed:
LPS-148882 Use proper name.

drewbrokke committed:
LPS-159104 account-admin-web: SF: uses a variable

cgoncas committed:
LPS-148882 We don't need to use a list, so we can avoid that conversion.

ozysouza committed:
LPS-146373 Use Navigator.gotoNavTab instead

ozysouza committed:
LPS-146373 Update locator

thaynam committed:
LPS-159917 fix align items

brianchandotcom committed:
LRCI-2853 prep next

ozysouza committed:
LPS-146373 If we have second verification click on X icon

brianchandotcom committed:
LPS-122228 Regen

brianchandotcom committed:
LPS-150378 prep next

brianchandotcom committed:
LPS-150378 prep next

brianchandotcom committed:
LPS-159022 upload

brianchandotcom committed:
LPS-148882 Inline

brianchandotcom committed:
LPS-148882 Make names the same

brianchandotcom committed:
LPS-159176 Wordsmith

brianchandotcom committed:
LPS-158149 and LPS-158655 sort mockHttpServletRequest.set*

brianchandotcom committed:
LPS-159877 Minor wordsmith

brianchandotcom committed:
LPS-159877 Regen
